samples/boards/hexiwear/4x4-buttons/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _hello_world:
```

Comment:
document labels must be unique across the entire documentation set.  I'm wondering if you made a copy of the hello_world sample to start writing the 4x4-buttons sample and then didn't update the content?